tools/tts/tts/tts.py

```diff
@@ -106,6 +106,10 @@ def tts_health_check():
 		return f"EXPIRED: {request_count}", 500
 	return f"OK: {request_count}", 200
 
+@app.route("/pitch-available")
+def pitch_available():
+	abort(500)
```

Comment:
eh, are you unable to implement pitch availability on the repo?

Reply:
pitch availability is dependent on RVC existing, since RVC is complicated to set up out the box and we don't have a default RVC model to ship that's using the repo voices, i set this up